cmd/object-handlers.go

```diff
@@ -48,16 +49,39 @@ func setGetRespHeaders(w http.ResponseWriter, reqParams url.Values) {
 	}
 }
 
+// getSourceIPAddress - get the source ip address of the request.
+func getSourceIPAddress(r *http.Request) string {
+	// Attempt to get ip from standard headers.
+	var ip string
+	for _, h := range []string{"X-Forwarded-For", "X-Real-Ip"} {
```

Comment:
Ok awesome (not the easy spoofing). I will change it to only support Real-Ip header. Thanks for the links.